ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -55,11 +56,12 @@ Verify Pod Toleration
 
 Verify RHODS User Groups
     [Documentation]    Verify User Configuration after the upgrade
-    [Tags]      Upgrade
-    ${admin}        Set Variable        ${payload[0]['spec']['groupsConfig']['adminGroups']}
-    ${user}     Set Variable        ${payload[0]['spec']['groupsConfig']['allowedGroups']}
-    Should Be Equal As Strings      '${admin}'      'rhods-admins,rhods-users'
-    Should Be Equal As Strings      '${user}'       'system:authenticated'
+    [Tags]      Upgrade     Operator
```

Comment:
I noticed the added tag `Operator` but this test won't work without running pre-upgrade test suite as it expects different values than default. We are setting these values in the pre-upgrade test suite here:
https://github.com/red-hat-data-services/ods-ci/blob/cb949633d30b38a7a524219aa5f611770fcd52a9/ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot#L78

Reply:
I will substitute by RHOAING tag

Reply:
I will replace by RHOAIENG tag